public/app/plugins/datasource/jaeger/CheatSheet.tsx

```diff
@@ -0,0 +1,58 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { useStyles2 } from '@grafana/ui';
+
+const linkColor = '#1F62E0';
+
+export default function CheatSheet() {
+  const styles = useStyles2(getStyles);
+  return (
+    <>
+      <h2 id="jaeger-cheat-sheet">Jaeger Cheat Sheet</h2>
+      <p>
+        Documentation for the Jaeger data source can be found{' '}
+        <a
+          href="https://grafana.com/docs/grafana/latest/datasources/jaeger"
+          target="_blank"
+          rel="noreferrer"
+          className={styles.linkColor}
+        >
+          here,
+        </a>{' '}
+        but here is a quick overview of the query types:
+      </p>
+
+      <hr />
+      <ul className={styles.removeBullets}>
```

Comment:
It's better to call this and `linkColor` something more descriptive as to the element it is styling not it's single function as it's function may change in time i.e. it may not always only need to removeBullets.